File: java/src/uk/co/datamantechnology/enigma/AllTests.java
package uk.co.datamantechnology.enigma;

import junit.framework.Test;
import junit.framework.TestSuite;

public class AllTests {

	public static Test suite() {
		TestSuite suite = new TestSuite(
				"Test for uk.co.datamantechnology.enigma");
		//$JUnit-BEGIN$
		suite.addTestSuite(AlphabetTest.class);
		suite.addTestSuite(RotorTest.class);
		suite.addTestSuite(EnigmaMachineTest.class);
		suite.addTestSuite(ConfigurationTest.class);
		suite.addTestSuite(SteckerTest.class);
		//$JUnit-END$
		return suite;
	}

}

File: python/src/all_tests.py
import unittest
import alphabetTest
import configurationTest
import enigmamachineTest
import steckerTest
import rotorTest


if __name__ == "__main__":
    result = unittest.TestResult()
    suite1 = alphabetTest.suite()
    suite2 = configurationTest.suite()
    suite3 = enigmamachineTest.suite()
    suite4 = rotorTest.suite()
    suite5 = steckerTest.suite()
    alltests = unittest.TestSuite([suite1, suite2, suite3, suite4, suite5])
    alltests.run(result)

    print("====================")
    print("Was successful: " + str(result.wasSuccessful()))
    print("\rTests run: " + str(result.testsRun))
